.gitignore

```diff
@@ -26,7 +26,7 @@ export/
 *.db
 *.wal
 *.parquet
-*.csv
+./*.csv
```

Comment:
This might have unintended side effects, consider including the specific path in the next section instead

Reply:
fixed